FordLabs/PeopleMover
# Update share access modal - part 2

## Issue
Resolves #95 - Break Share Access out into two modals

## What was done
- [x] Updated share access modal pop-up to have a two modal design
- [x] Improved accessibility of the modal
- [x] Added test coverage for share access modal   

## How to test
1. Open a space
2. Open the account dropdown
3. Select "Share Access"
4. Observe the new layout of the share access modal
5. Compare to Figma designs
6. Tab through modal with screenreader
